AMW_angular/io/src/app/core/amw-constants.ts

```diff
@@ -1,5 +1,5 @@
 export const AMW_LOGOUT_URL = 'amw.logoutUrl';
 // used for moment.js
-export const DATE_FORMAT = 'DD.MM.yyyy HH:mm';
+export const DATE_FORMAT = 'yyyy-MM-dd_HHmm';
 // use by angular date pipes (unicode): https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table
 export const DATE_FORMAT_ANGULAR = 'dd.MM.yyyy HH:mm';
```

Comment:
I think DATE_FORMAT_ANGULAR is no longer required and can be removed

Reply:
It is used in auditview-table.service.ts but only once.

Reply:
Can you please refactor it so that only `DATE_FORMAT` is used and delete `DATE_FORMAT_ANGULAR`?